### PR TITLE
Add an security.txt for joomla.org

### PR DESCRIPTION
**Pull request by zero-24**
For details see: https://securitytxt.org/ & https://tools.ietf.org/html/draft-foudil-securitytxt-09

### CODE REVIEW

File: .well-known/security.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+-----BEGIN PGP SIGNED MESSAGE-----
+Hash: SHA512
+
+Contact: security@joomla.org
+Encryption: https://developer.joomla.org/security/gpg-keys.html
+Acknowledgments: https://developer.joomla.org/security-centre.html
+Preferred-Languages: en
+Canonical: https://joomla.org/.well-known/security.txt
```
Comment (mbabker) [inline .well-known/security.txt]: The canonical address for .org is www.joomla.org, non-prefixed is redirected.

**Comment by zero-24** [comment]
Thanks @mbabker fixed. Chrome fooled me on this by hiding the www.

**Comment by zero-24** [comment]
@conconnl please merge when ok on your side as discussed this has been uploaded to the main site already.

**Comment by mbabker** [comment]
> Chrome fooled me on this by hiding the www.

This is why the browsers moving to hide subdomains like m. and www. is bad for business.

**Comment by zero-24** [comment]
Thanks. Chrome even removed the option to add that stuff back lately :(

**Pull request merged**